src/securityPolicy.c

```diff
@@ -263,7 +273,41 @@ security_policy_t policyForSignTxOutputAddressParams(
 	SHOW_UNLESS(has_reasonable_account_and_address(&params->spendingKeyPath));
 	SHOW_UNLESS(is_standard_base_address(params));
 
-	ALLOW_IF(true);
+	ALLOW();
+}
+
+// multiasset token groups in outputs
+security_policy_t policyForSignTxOutputAssetGroup(
```

Comment:
@janmazak this has the unintended side-effect of showing tokens for change outputs which should otherwise be hidden